### PR TITLE
Wip tutorial

### PR DESCRIPTION
**Pull request by deniznida**

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @jmburges

File: lib/operations.rb [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-	elsif 
-		speed < 40
-		true
-	else
-		false
-	end
+  if speed > 60 
+    true
+  elsif 
+    speed < 40
```
Comment (jmburges) [inline lib/operations.rb]: hmmm why are you putting the condition on a new line?

File: TUTORIAL.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Guide to Solving Operators
+
+### `unsafe?`
+
+After running `rspec` for the first time we see we have seven tests failing.
+
```
Comment (jmburges) [inline TUTORIAL.md]: This is super early in the semester. Can you show the failing tests and then explain them a bit? Like how did you go from failing test to that method below?

Comment (jmburges) [inline TUTORIAL.md]: Feel free to also include "debugging" techniques. Like copy pasting the method into irb or using binding.pry to play with stuff. @SophieDeBenedetto do they know pry yet when operators come up?

**Comment by jmburges** [comment]
Few issues. It's a super early lab so we need to take very little for granted.

**Comment by jmburges** [comment]
Looks good! :shipit:

**Pull request merged**